### DISCUSSION TITLE
In server mode, can we stop generating (like in chat UI there's stop generating button)

### DISCUSSION THREAD

**Discussion by x4080** [ggml-org/llama.cpp#4721] [Q&A]
In server mode, can we stop generating (like in chat UI there's stop generating button)

Any idea ?

Thanks

**Comment by shibe2** [comment]
When using streaming completion, close the HTTP connection. It stops generation, but it doesn't seem to stop prompt processing.

**Reply by teleprint-me** [reply]
> It stops generation, but it doesn't seem to stop prompt processing.

Yeah, this is something I ran into as well. It is possible to interrupt it completely. Not sure how to signal that in the browser.

**Comment by x4080** [comment]
@shibe2 Thanks
